src/Handler/ImageHandler.php

```diff
@@ -259,7 +259,11 @@ private function updateMediaMetaData(FilesystemInterface $filesystem): void
     {
         $file = $this->mediaPath . $this->media->basePath() . $this->media->filename();
 
-        $imageData = \getimagesizefromstring($filesystem->read($file));
+        try {
```

Comment:
After a brief discussion: this should be handled within the command. The general idea to make it not fail while processing images just because one file might've gone missing is ok.
This applies to the change in `ImageProcessor` as well